src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--29 calendar--start-day--sun">
+      <div class="day--1 day"></div>
```

Comment:
Day is element of the calendar. Not clear from naming. Violates BEM convention.

Reply:
Here the day can be a block. It dosen't have any properties for positon